idea_town/frontend/static-src/app/views/header-view.js

```diff
@@ -30,20 +29,15 @@ export default BaseView.extend({
   //       to pick up csrftoken cookie changes.
   logout(evt) {
     evt.preventDefault();
-    xhr({
+    fetch(app.getDocHeadLink('x-logout', 'href'), {
```

Comment:
> I'm not totally convinced we need to abstract out the account-related urls

It starts feeling to me like trouble to hard-code URLs anywhere, and try to use Django's reverse URL lookup. It's bitten me rarely, but it has bitten me. Don't Repeat Yourself, magic numbers, and all that.

> Random questions: Exactly which kinds of data do you want to specify in the DOM? Just enough to get the page connected to the API server? Is one x-app-startup tag, containing a serialized object, cleaner than a bunch of individual tags & running to the DOM over and over?

Basically, I don't really have a plan in mind. That spot in the DOM looked like a less-terrible spot to stick a few shared URLs and strings than the inline script.

Figured we could iterate on that as things come up that the app needs from the server. I can go back to the drawing board and try coming up with more of a startup data API resource. Might be worth a follow up issue though

> Something to think about for the future: it seems weird that we get the logout URI via app.getDocHeadLink, but the login URI (which contains a redirect endpoint, too) is just hard-coded in the view. 

I'd missed the login link in that view. I'd want to change that, too. 

> Also seems like we might want to put the csrftoken (app.me.csrfToken) in the same place inside the app as the endpoints?

We get that from a cookie right now. Do you mean, it should get included in an API resource too?